README.md

```diff
@@ -56,15 +56,21 @@ GitHub Enterprise repositories are not supported.
 
 ![image](/images/install02.png)
 
-3) Then click "Add tool".
+3) Depending on whether or not the repo already has a scanning tool configured, you will see "Add tool" or "Configure scanning tool". If you see "Add tool" click it and skip step 3b. If you see "Configure scanning tool" proceed to step 3b.
```

Comment:
As you'll see in the commit below, I'd suggest using generic references to numbered steps within the text body so that way, if we add more steps in the future, we won't need to revise all the references.

```suggestion
3) Depending on whether the repository already has a scanning tool configured, you will see "Add tool" or "Configure scanning tool". If you see "Add tool" click it and skip step `b`. If you see "Configure scanning tool" proceed to step `b`.
```

Reply:
>  if we add more steps in the future, we won't need to revise all the references.

Personally, I think one benefit of markdown is the ability to read it still even when not rendered by GitHub or other platforms. So I'm on the fence about seeing a bunch of repeated 1's, but I don't claim to know markdown best practices.

Reply:
That's fair — it's more of a stylistic preference than a best practice.